src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -89,12 +90,32 @@ public function getTranslationNodeVisitor(): TranslationNodeVisitor
         return $this->translationNodeVisitor ?: $this->translationNodeVisitor = new TranslationNodeVisitor();
     }
 
-    public function trans(string $message, array $arguments = [], string $domain = null, string $locale = null, int $count = null): string
+    public function trans(string $message, array $arguments = [], string $domain = null, string $locale = null, int $count = null, ?string $default = null): string
     {
         if (null !== $count) {
             $arguments['%count%'] = $count;
         }
 
-        return $this->getTranslator()->trans($message, $arguments, $domain, $locale);
+        $translated = $this->getTranslator()->trans($message, $arguments, $domain, $locale);
+
+        if (null === $default) {
+            return $translated;
+        }
+
+        if (!$this->translator instanceof TranslatorBagInterface) {
```

Comment:
I would have expected something simpler like: `return $this->trans($default, $arguments, $domain, $locale);`

Reply:
I don't understand. You must know if the message is is defined before fallbacking or not.

Reply:
Just realized that this is done in Twig only. Why? Wouldn't it make sense to support this on the main PHP translator?

Reply:
Why: Because I was super extra lazy (need to add many deprecation, bc layer, etc). 
But OK, I will add to the translator component :)